tests/subsys/settings/fcb/src/settings_test_fcb.c

```diff
@@ -313,6 +313,20 @@ int c3_handle_export(int (*cb)(const char *name, void *value, size_t val_len))
 	return 0;
 }
 
+void tests_settings_check_target(void)
+{
+	const struct flash_area *fap;
+	int rc;
+	u8_t wbs;
+
+	rc = flash_area_open(DT_FLASH_AREA_STORAGE_ID, &fap);
+	zassert_true(rc == 0, "Can't open storage flash area");
+
+	wbs = flash_area_align(fap);
+	zassert_true(wbs <= 16,
+		"Flash driver is not compatible with the settings fcb-backend");
```

Comment:
ok, so it means test will work with any driver with wbs 4, 8, 16?

Reply:
Yes. And 16 is limitation by fcb-backend.